manifests/s/stacklok/minder/0.0.48/stacklok.minder.locale.en-US.yaml

```diff
@@ -0,0 +1,130 @@
+  * cf38dc4e804e15de852ac996f7a30f8b3d91bec7: build(deps): bump golang.org/x/term from 0.19.0 to 0.20.0 (#3241) (@dependabot[bot])
+  * 6849a9356d4d87df0b4bcb3a090255ad35ae4cd7: build(deps): bump golang.org/x/tools from 0.20.0 to 0.21.0 in /tools (#3258) (@dependabot[bot])
+  * 0ab2261a06cba940c96c83b25f201e40d11d61c1: build(deps): bump golangci/golangci-lint-action from 4.0.0 to 5.0.0 (#3172) (@dependabot[bot])
+  * 30a63bb03ff03d129c08f85e8eea8f0c6fc10d1e: build(deps): bump golangci/golangci-lint-action from 5.0.0 to 5.1.0 (#3206) (@dependabot[bot])
+  * 727de7db937f87e9392cfd0ea20a7fc0e939e887: build(deps): bump golangci/golangci-lint-action from 5.1.0 to 5.3.0 (#3242) (@dependabot[bot])
+  * 09bed2fb5727c6b44d423dcf324a0a1f6e45ee60: build(deps): bump golangci/golangci-lint-action from 5.3.0 to 6.0.0 (#3255) (@dependabot[bot])
+  * 7c50e6522f3bcb2afaa28819ea286ba566722a4e: build(deps): bump google.golang.org/protobuf from 1.33.0 to 1.34.0 (#3217) (@dependabot[bot])
+  * ca981c535bb2cd6e465dec287ee4486cbcf3ab08: build(deps): bump google.golang.org/protobuf from 1.33.0 to 1.34.0 in /tools (#3216) (@dependabot[bot])
+  * 78e58d8dce39fa26b5670f52f374629fd51fe389: build(deps): bump google.golang.org/protobuf from 1.34.0 to 1.34.1 (#3254) (@dependabot[bot])
+  * e3c502650be42b007505d9fc7f715ff9a0ef26ba: build(deps): bump google.golang.org/protobuf from 1.34.0 to 1.34.1 in /tools (#3257) (@dependabot[bot])
+  * 8421c825dfc93f9c829d1e610b21c24352f5fdee: build(deps): bump k8s.io/apimachinery from 0.29.4 to 0.30.0 (#3126) (@dependabot[bot])
+  * 7a335b75316fc43698845da1dc93a407853d55f1: build(deps): bump k8s.io/client-go from 0.29.3 to 0.29.4 (#3121) (@dependabot[bot])
+  * 2e88d5af84ac30bbf408bf4cd45b185953715966: build(deps): bump k8s.io/client-go from 0.29.4 to 0.30.0 (#3125) (@dependabot[bot])
+  * 4ebde7f2b0f79c36ee2fdbe09a952e096e5eeb10: build(deps): bump peter-evans/create-pull-request from 6.0.3 to 6.0.4 (#3127) (@dependabot[bot])
+  * 640e29fa5be10737e139ee12936f1ea54e3c24b4: build(deps): bump peter-evans/create-pull-request from 6.0.4 to 6.0.5 (#3186) (@dependabot[bot])
+  * ccfd735e0a93e806a9965fe1ac0e2086249e0e81: build(deps): bump react from 18.2.0 to 18.3.0 in /docs (#3179) (@dependabot[bot])
+  * e63534ca3b78f4105e5e722ea28b6f0ee0e0d810: build(deps): bump react from 18.3.0 to 18.3.1 in /docs (#3195) (@dependabot[bot])
+  * 98d22a4e0da70d0362f393ad24057e9e270d5a4c: build(deps): bump react-dom from 18.2.0 to 18.3.1 in /docs (#3198) (@dependabot[bot])
+  * 7e191c5378ef15716be1c89cc3ae31c6bb34b1b1: build(deps): bump sigs.k8s.io/release-utils from 0.7.7 to 0.8.1 (#3166) (@dependabot[bot])
+  * e1b25dbf50e6e7926a1f7df12ed19424ae5c3d5d: build(deps): bump slsa-framework/slsa-github-generator from 1.10.0 to 2.0.0 (#3146) (@dependabot[bot])
+  * 6528f665f15846c561ce5460df16b8cf0db42701: build(deps): bump styled-components from 6.1.8 to 6.1.9 in /docs (#3215) (@dependabot[bot])
```

Comment:
```suggestion
  * Add Helm configuration for feature flags (#3188) (@evankanderson)
  * Add ProviderManager, make provider deletion generic (#3162) (@dmjb)
  * Add `reminder` service with empty sendReminders logic (#2638) (@Vyom-Yadav)
  * Add docs for using feature flags when developing Minder (#3189) (@evankanderson)
  * Add flag telling if repo from provider is already registered (#3209) (@blkt)
  * Address review comments for labels filtering (#3137) (@jhrozek)
  * Allow empty provider in repo reconcile command (#3231) (@eleftherias)
  * Allow full profile updates through the PATCH handler (#2990) (@jhrozek)
  * Allow setting log level for mindev ruletype test (#3119) (@jhrozek)
  * Artifact: Unify signer identity URIs (#3238) (@puerco)
  * Bug: Remove impossible condition (#3213) (@puerco)
  * Bump docusaurus from 3.2.1 to 3.3.2 (#3248) (@eleftherias)
  * Bump go base images to go 1.22.2 (#3161) (@dmjb)
  * Bump sigstore-go to v0.3.0 and add local registry for tests (#3154) (@puerco)
  * Delete docs/docs/how-to/using-minder-with-ghas.md (#3204) (@meganbruce)
  * Distinguish between score 0 and missing score in Trusty (#3234) (@jhrozek)
  * Don't trim path from builder URI (#3214) (@puerco)
  * Don't use SELECT * when joining profile tables (#3130) (@jhrozek)
  * Fix bug handling images with slashes in ref (#3211) (@puerco)
  * Fix goreleaser dependencies. (#3261) (@blkt)
  * Fix repo deletion (#3140) (@jhrozek)
  * Fix webhook secret file names (#3201) (@jhrozek)
  * Implement GitHubClientFactory (#3203) (@dmjb)
  * Implement ProviderFactory and ProviderClassFactory (#3131) (@dmjb)
  * Implement `CanImplement` method for providers (#3115) (@dmjb)
  * Initial implementation of IDP interface and Keycloak implementation (#3155) (@evankanderson)
  * Make Project Create/Delete a service (#3227) (@dmjb)
  * Make provider class mandatory in DB (#3132) (@dmjb)
  * Merge pull request from GHSA-9c5w-9q3f-3hv7 (@JAORMX)
  * More descriptive error message when validating secrets with any of the fallback webhooks fails (#3176) (@jhrozek)
  * Parse correctly github uris, dont use cert ext uri (#3250) (@puerco)
  * Pass RestClientCache as constructor param (#3222) (@dmjb)
  * Re-enable go CoverageRedesign (#3252) (@prezha)
  * Read the webhook secret from a file (#3175) (@jhrozek)
  * Refactor application wireup (#3230) (@dmjb)
  * Refactor repo deletion to move db/provider logic behind interface (#3200) (@dmjb)
  * Remove GitHub discussions from README (#3129) (@eleftherias)
  * Remove OpenFGA CVE ignore from `.trivyignore` file (#3228) (@JAORMX)
  * Remove last CVE from `.trivyignore` (#3229) (@JAORMX)
  * Remove provider name from event wrapper (#3139) (@dmjb)
  * Remove use of ProviderBuilder by repo handler (#3224) (@dmjb)
  * Return 500 if Github AppName is empty when enrolling provider (#3124) (@dmjb)
  * Return verified attestation in verification results (#3212) (@puerco)
  * Skip or error when trying to register an archived repository (#3135) (@rdimitrov)
  * Surface bundle fetch error if gh client is not present (#3249) (@JAORMX)
  * Update CONTRIBUTING.md (#3251) (@dussab)
  * Update README.md (#3118) (@eryn-muetzel)
  * Update index.md (#3117) (@eryn-muetzel)
  * Use ProviderManager in reconciler (#3232) (@dmjb)
  * Use ProviderManager in webhook handler (#3202) (@dmjb)
  * Use `go generate` directives for mock generation (#3159) (@dmjb)
  * Use provider ID instead of name when sending events (#3093) (@dmjb)
  * Use webhook secrets from files (#3177) (@jhrozek)
  * Warn about empty secret and skip the update in the webhook updater tool (#3208) (@jhrozek)
  * Workaround: Remove `MINDER_FLAGS_GO_FEATURE_FILE_PATH` env variable from helm deployment (#3260) (@JAORMX)
  * build(deps): bump actions/checkout from 4.1.2 to 4.1.3 (#3141) (@dependabot[bot])
  * build(deps): bump actions/checkout from 4.1.3 to 4.1.4 (#3171) (@dependabot[bot])
  * build(deps): bump actions/checkout from 4.1.4 to 4.1.5 (#3256) (@dependabot[bot])
  * build(deps): bump actions/setup-go from 5.0.0 to 5.0.1 (#3233) (@dependabot[bot])
  * build(deps): bump anchore/sbom-action from 0.15.10 to 0.15.11 (#3197) (@dependabot[bot])
  * build(deps): bump bufbuild/buf-setup-action from 1.30.1 to 1.31.0 (#3156) (@dependabot[bot])
  * build(deps): bump clsx from 2.1.0 to 2.1.1 in /docs (#3144) (@dependabot[bot])
  * build(deps): bump github.com/bufbuild/buf from 1.30.1 to 1.31.0 in /tools (#3158) (@dependabot[bot])
  * build(deps): bump github.com/docker/docker from 26.0.0+incompatible to 26.0.2+incompatible in /tools (#3133) (@dependabot[bot])
  * build(deps): bump github.com/docker/docker from 26.0.1+incompatible to 26.0.2+incompatible (#3134) (@dependabot[bot])
  * build(deps): bump github.com/go-playground/validator/v10 from 10.19.0 to 10.20.0 (#3205) (@dependabot[bot])
  * build(deps): bump github.com/golang-migrate/migrate/v4 from 4.17.0 to 4.17.1 (#3136) (@dependabot[bot])
  * build(deps): bump github.com/golangci/golangci-lint from 1.57.2 to 1.58.0 in /tools (#3243) (@dependabot[bot])
  * build(deps): bump github.com/open-feature/go-sdk-contrib/providers/go-feature-flag from 0.1.35 to 0.1.36 (#3192) (@dependabot[bot])
  * build(deps): bump github.com/open-policy-agent/opa from 0.63.0 to 0.64.1 (#3191) (@dependabot[bot])
  * build(deps): bump github.com/openfga/cli from 0.3.0 to 0.3.1 in /tools (#3207) (@dependabot[bot])
  * build(deps): bump github.com/openfga/go-sdk from 0.3.5 to 0.3.6 (#3218) (@dependabot[bot])
  * build(deps): bump github.com/openfga/openfga from 1.5.1 to 1.5.3 (#3122) (@dependabot[bot])
  * build(deps): bump github.com/signalfx/splunk-otel-go/instrumentation/github.com/lib/pq/splunkpq from 1.15.0 to 1.16.0 (#3190) (@dependabot[bot])
  * build(deps): bump github.com/styrainc/regal from 0.20.1 to 0.21.0 (#3157) (@dependabot[bot])
  * build(deps): bump github.com/styrainc/regal from 0.21.0 to 0.21.3 (#3193) (@dependabot[bot])
  * build(deps): bump github.com/thomaspoignant/go-feature-flag from 1.26.0 to 1.27.0 (#3240) (@dependabot[bot])
  * build(deps): bump github/codeql-action from 3.25.0 to 3.25.1 (#3128) (@dependabot[bot])
  * build(deps): bump github/codeql-action from 3.25.1 to 3.25.2 (#3147) (@dependabot[bot])
  * build(deps): bump github/codeql-action from 3.25.2 to 3.25.3 (#3185) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc from 0.50.0 to 0.51.0 (#3184) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp from 0.50.0 to 0.51.0 (#3169) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/otel/exporters/prometheus from 0.47.0 to 0.48.0 (#3181) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/otel/exporters/stdout/stdouttrace from 1.25.0 to 1.26.0 (#3183) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/otel/sdk/metric from 1.25.0 to 1.26.0 (#3182) (@dependabot[bot])
  * build(deps): bump go.opentelemetry.io/otel/trace from 1.25.0 to 1.26.0 (#3170) (@dependabot[bot])
  * build(deps): bump golang.org/x/crypto from 0.22.0 to 0.23.0 (#3253) (@dependabot[bot])
  * build(deps): bump golang.org/x/oauth2 from 0.19.0 to 0.20.0 (#3239) (@dependabot[bot])
  * build(deps): bump golang.org/x/term from 0.19.0 to 0.20.0 (#3241) (@dependabot[bot])
  * build(deps): bump golang.org/x/tools from 0.20.0 to 0.21.0 in /tools (#3258) (@dependabot[bot])
  * build(deps): bump golangci/golangci-lint-action from 4.0.0 to 5.0.0 (#3172) (@dependabot[bot])
  * build(deps): bump golangci/golangci-lint-action from 5.0.0 to 5.1.0 (#3206) (@dependabot[bot])
  * build(deps): bump golangci/golangci-lint-action from 5.1.0 to 5.3.0 (#3242) (@dependabot[bot])
  * build(deps): bump golangci/golangci-lint-action from 5.3.0 to 6.0.0 (#3255) (@dependabot[bot])
  * build(deps): bump google.golang.org/protobuf from 1.33.0 to 1.34.0 (#3217) (@dependabot[bot])
  * build(deps): bump google.golang.org/protobuf from 1.33.0 to 1.34.0 in /tools (#3216) (@dependabot[bot])
  * build(deps): bump google.golang.org/protobuf from 1.34.0 to 1.34.1 (#3254) (@dependabot[bot])
  * build(deps): bump google.golang.org/protobuf from 1.34.0 to 1.34.1 in /tools (#3257) (@dependabot[bot])
  * build(deps): bump k8s.io/apimachinery from 0.29.4 to 0.30.0 (#3126) (@dependabot[bot])
  * build(deps): bump k8s.io/client-go from 0.29.3 to 0.29.4 (#3121) (@dependabot[bot])
  * build(deps): bump k8s.io/client-go from 0.29.4 to 0.30.0 (#3125) (@dependabot[bot])
  * build(deps): bump peter-evans/create-pull-request from 6.0.3 to 6.0.4 (#3127) (@dependabot[bot])
  * build(deps): bump peter-evans/create-pull-request from 6.0.4 to 6.0.5 (#3186) (@dependabot[bot])
  * build(deps): bump react from 18.2.0 to 18.3.0 in /docs (#3179) (@dependabot[bot])
  * build(deps): bump react from 18.3.0 to 18.3.1 in /docs (#3195) (@dependabot[bot])
  * build(deps): bump react-dom from 18.2.0 to 18.3.1 in /docs (#3198) (@dependabot[bot])
  * build(deps): bump sigs.k8s.io/release-utils from 0.7.7 to 0.8.1 (#3166) (@dependabot[bot])
  * build(deps): bump slsa-framework/slsa-github-generator from 1.10.0 to 2.0.0 (#3146) (@dependabot[bot])
  * build(deps): bump styled-components from 6.1.8 to 6.1.9 in /docs (#3215) (@dependabot[bot])
```